react-components/src/components/RuleBasedOutputs/RuleBasedOutputsSelector.tsx

```diff
@@ -106,19 +118,16 @@ async function initializeRuleBasedOutputs({
   timeseriesDatapoints
 }: {
   model: CogniteCadModel;
-  assetMappings: InfiniteData<ModelMappingsWithAssets[]>;
+  assetMappings: AssetMapping3D[];
```

Comment:
Just to be sure, I'd like @danpriori to approve that this is okay :sweat_smile:

Reply:
Hmmm I think all of these changes will affect the changes I've made in the PR

Among a couple of changes, I already removed the request for asset mappings from the hook 
`useAllMappedEquipmentAssetMappings`. I changed to the `useAssetMappedNodesForRevisions` and with that a couple of other changes were made and affected by this and other improvements for caching with rule based. And it is not using the pagination as is getting from that hook now.

Although some of these changes in this PR are aligned with the changes I made in the Rule Based PR.

See it here: https://github.com/cognitedata/reveal/pull/4652/files#

So what you guys think is it better to do then?

Reply:
This is not affecting any changes to rule based coloring